streaming/src/main/scala/org/apache/spark/streaming/scheduler/StreamingListenerForwardingBus.scala

```diff
@@ -17,17 +17,38 @@
 
 package org.apache.spark.streaming.scheduler
 
-import java.util.concurrent.atomic.AtomicBoolean
+import org.apache.spark.scheduler.{LiveListenerBus, SparkListener, SparkListenerEvent}
+import org.apache.spark.util.ListenerBus
 
-import org.apache.spark.Logging
-import org.apache.spark.util.AsynchronousListenerBus
+/**
+ * Wrap StreamingListenerEvent as SparkListenerEvent so that it can be posted to Spark listener bus.
+ */
+private[streaming] case class WrappedStreamingListenerEvent(
```

Comment:
I am inclined to put this as an internal class of `StreamingListenerForwardingBus` because this is basically an internal implementation detail that streaming events are wrapped as spark events.